source/wp-content/themes/wporg-news-2021/sass/base/_normalize.scss

```
@@ -1,14 +1,16 @@
-
-// Remove the margin in all browsers.
 body {
+	// Remove the margin in all browsers.
 	margin: 0;
 	padding: 0;
-}
 
-// Smooth out the fonts
-body {
+	// Smooth out the fonts
 	-webkit-font-smoothing: antialiased;
 	-moz-osx-font-smoothing: grayscale;
+
+	// Blocks, by default, have a top margin. This is annoying.
```

```suggestion
	// Blocks, by default, have a top margin, but we usually only want that in post content. When something outside post content needs a margin, it usually needs to be custom anyway, rather than exactly matching `--wp--style--block-gap`.
```

I think this describes what you were implying, but feel free to make it more accurate if I misunderstood. I agree it's annoying, I just want us to remember _why_ when we come back to this in 6 months.